lua/nvim-lsp-installer/servers/zk/init.lua

```diff
@@ -0,0 +1,18 @@
+local server = require "nvim-lsp-installer.server"
+local std = require "nvim-lsp-installer.installers.std"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        homepage = "https://https://github.com/mickael-menu/zk",
+        languages = { "markdown" },
+        installer = std.ensure_executables {
```

Comment:
Any reason you decided not to actually install zk here? It seems like it's pretty straight forward, they even provide prebuilt binaries?

This approach of only checking whether the executables exist is only applied in rare cases where it doesn't make much sense to install the LSP locally. Dart and Sourcekit (Swift) are the only two so far - reason being they both ship the LSP server as part of their SDK; so all we do is provide instructions on how to install these separately. This doesn't seem to be the case here?